src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -226,7 +227,7 @@ public void Start()
             this.WalletTipHash = this.LastReceivedBlockHash();
 
             // Save the wallets file every 5 minutes to help against crashes.
-            this.asyncLoop = this.asyncLoopFactory.Run("Wallet persist job", token =>
+            this.asyncLoop = this.asyncProvider.CreateAndRunAsyncLoop("Wallet persist job", token =>
```

Comment:
Come to think about it the background work for saving the wallet should go in the feature (not this PR obviously)